src/components/Contact/Contact.tsx

```diff
@@ -40,15 +40,19 @@ export const Contact = ({
     [`${className}`]: className
   });
 
+  console.log(imageSources);
```

Comment:
Remove the console log (unless this is needed for testing)

Reply:
tbh this shouldn't get past the linter in the pre-commit hook, I'll double-check that and see if we can ensure this doesn't happen again

Reply:
Don't know why I keep doing it. Removed it.